python/lsst/daf/persistence/butlerSubset.py

```diff
@@ -238,7 +242,10 @@ def subItems(self, level=None):
         """
 
         if level is None:
-            mappers = self.butlerSubset.butler.repository.mappers()
+            mappers = []
+            for repoData in self.butlerSubset.butler.inputs + self.butlerSubset.butler.outputs:
+                if repoData.repo._mapper not in mappers:
+                    mappers.append(repoData.repo._mapper)
             if len(mappers) != 1:
                 raise RuntimeError("Support for multiple repositories not yet implemented!")
```

Comment:
At least this should say "Support for multiple mappers", since this PR is about multiple repositories :smiley:.  It might be possible to use multiple mappers if they all had the same default level.  (I'm a little concerned about the "todo" comment below, as well.)